src/index.ts

```diff
@@ -1,2 +1,6 @@
-export { fromEvent, fromFileHandles } from "./file-selector.js";
+export {
```

Comment:
I do believe that providing a convenience function that runs though each would help users migrate smoothly.

Reply:
I am going to rename and refactor `fromEvent` to something else as well, perhaps we can then evaluate adding it back and see if it is worth to keep it in a deprecated manner?

I have to do some digging as well and see how we're going to go about upgrading this dependency in the main project, so there will likely be a `beta` branch there as well.